packages/devtools_app_shared/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ dependencies:
   logging: ^1.1.1
   meta: ^1.9.1
   pointer_interceptor: ^0.9.3+3
-  vm_service: ^13.0.0
+  vm_service: ^14.0.0
```

Comment:
add an entry to the CHANGELOG files for both devtools_app_shared and devtools_extensions. Bump the version for these packages too.

Do we also need to bump the vm_service version in devtools_shared or is that fine to stay as a range?

Reply:
Ah, oops. I forgot to update those. Done.

The constraint in `package:devtools_shared` can stay as they are.